tests/ResourceSyncingTest.php

```diff
@@ -126,6 +127,244 @@
     ], ResourceUser::first()->getAttributes());
 });
 
+test('creating the resource in tenant database creates it in central database as 1:1 copy when creation attributes are not specified', function () {
+    // Assert no user exists in central DB
+    expect(ResourceUser::all())->toHaveCount(0);
+
+    $tenant = ResourceTenant::create();
+    migrateTenantsResource();
+
+    tenancy()->initialize($tenant);
+
+    // Create the user in tenant DB
+    ResourceUser::create([
+        'global_id' => 'acme',
+        'name' => 'John Doe',
+        'email' => 'john@localhost',
+        'password' => 'secret',
+        'role' => 'commenter', // unsynced
+    ]);
+
+    tenancy()->end();
+
+    // Assert central user and Resource user has exact same attributes and values
+    expect(CentralUser::first()->toArray())->toEqual(ResourceUser::first()->toArray());
+});
+
+test('creating the resource in tenant database creates it in central database with default attributes values', function () {
+    // override method in ResourceUser class to return attribute default values
+    class ResourceUserWithDefaultValues extends ResourceUser {
+        public function getResourceCreationAttributes(): array
```

Comment:
We don't define classes inside of tests. They should be at the end of the test file.